### DISCUSSION TITLE
about-the-author/csdn-1000wan

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#6] [Announcements]
# about-the-author/csdn-1000wan

一份通俗易懂、风趣幽默的Java学习指南，内容涵盖Java基础、Java并发编程、Java虚拟机、Java企业级开发、Java面试等核心知识点。学Java，就认准Java程序员进阶之路

https://tobebetterjavaer.com/about-the-author/csdn-1000wan.html

**Comment by rao548** [comment]
二哥牛逼，2022 我们一起飞~~

**Reply by itwanger** [reply]
好滴，让我们红尘作伴过得潇潇洒洒！

**Comment by wyxazgye** [comment]
二哥牛逼，2022 我们一起飞~~

**Reply by itwanger** [reply]
来吧，还有半年，一起卷
